src/containers/navbar-left/navbar-left.js

```diff
@@ -25,16 +25,13 @@ const NavbarLeft = () => {
     }
   };
 
-  const CategoriesView = () => {
-    if (!categories) {
-      return null;
-    }
-    return categories.map(({ _id, name }) => (
+  const categoriesList = !categories
```

Comment:
Usually, it is better to have positive case first `categories ? categories.map(...) : null`

Reply:
Changed